.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+.nyc_output
```

Comment:
Added this here because of adding the `nyc` code coverage library. Maybe we want to keep it and check it in to use in CI or in a README? I need to read more about it

Reply:
seems like its recommended to be ignored (at least it is here) https://stackoverflow.com/questions/50960312/should-nyc-output-be-included-in-gitignore

looks like its stored there so nyc report commands can be ran at a later point https://github.com/istanbuljs/nyc/issues/197

Reply:
Sounds good. I'll leave it ignored for now